{"name": "test_login_with_valid_email_and_invalid_password", "status": "passed", "start": 1710934865480, "stop": 1710934868551, "uuid": "fc2df6d5-6fa0-4eac-ac58-1fcf987788de", "historyId": "a87129e8bc488f2c2b04474f72c047f1", "testCaseId": "a87129e8bc488f2c2b04474f72c047f1", "fullName": "tests.test_login.TestLogin#test_login_with_valid_email_and_invalid_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "CTIVL0145"}, {"name": "thread", "value": "11060-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}